src/recipient_dashboard/templates/recipient_dashboard/dashboard.html

```diff
@@ -1,90 +1,137 @@
-                                               href="https://www.google.com/maps/dir/?api=1&destination={{ donation.organization.address|urlencode }}"
-                                               target="_blank"
-                                               rel="noopener noreferrer">
-                                                {{ donation.organization.organization_name }}
-                                                <br>
-                                                {{ donation.organization.address }}
-                                            </a>
-                                        </div>
-                                        <div class="column is-one-third has-text-centered">
-                                            <strong>Pickup by:</strong>
-                                            <p>{{ donation.pickup_by }}</p>
-                                        </div>
-                                    </div>
-                                </div>
+                                    href="https://www.google.com/maps/dir/?api=1&destination={{ donation.organization.address|urlencode }}"
+                                    target="_blank" rel="noopener noreferrer">
+                                    {{ donation.organization.organization_name }}
+                                    <br>
+                                    {{ donation.organization.address }}
+                                </a>
+                            </div>
+                            <div class="column is-one-third has-text-centered">
+                                <strong>Pickup by:</strong>
+                                <p>{{ donation.pickup_by }}</p>
                             </div>
-                            <footer class="card-footer">
-                                <a href="{% url 'reserve_donation' donation.donation_id %}"
-                                   class="card-footer-item">Reserve</a>
-                            </footer>
                         </div>
                     </div>
-                {% endfor %}
+                </div>
+                <footer class="card-footer">
+                    <a href="{% url 'reserve_donation' donation.donation_id %}" class="card-footer-item">Reserve</a>
+                </footer>
             </div>
-        {% else %}
-            <p class="box">No donations available at the moment.</p>
-        {% endif %}
+        </div>
+        {% endfor %}
     </div>
-    <script>
-        // Auto-dismiss notification after 10 seconds
-        document.addEventListener('DOMContentLoaded', () => {
-            const notifications = document.querySelectorAll('.notification');
-            notifications.forEach(notification => {
-                setTimeout(() => {
-                    notification.style.display = 'none';
-                }, 10000);
-            });
+    {% else %}
+    <p class="box">No donations available at the moment.</p>
+    {% endif %}
+</div>
+<script>
```

Comment:
Can you please shift this into a separate JS file so that it is easy to maintain and reuse static components?

Reply:
Since these notifications are used by all the views, we can have it under notifications.js and globally reference/link the script

Reply:
Good idea, thanks Arsalan!